### PR TITLE
Added r/riprequest along with blockthespot

### PR DESCRIPTION
**Pull request by Father-Dimi**
decided to stop being lazy and added them

### CODE REVIEW

**Comment by Father-Dimi** [comment]
also added vimms lair to rom

**Comment by kekkodance** [comment]
you stoopid vimms "liar" is already in roms

**Comment by kekkodance** [comment]
also fix your gremmer

**Comment by doggo404** [comment]
> you stoopid vimms "liar" is already in roms

calm down fool

File: Music/music.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ description: List of sources for getting all genres of Music.
 
 [**Unreleased Disco**](https://unreleased.me/) - Huge archive of Unreleased Hip-Hop/Pop Songs from various Artists.
 
+[**R/RipRequest**](https://www.reddit.com/r/riprequests/) - A sub-reddit for requesting songs (most flacs) **_Is currently private but you can request access_**
```
Comment (doggo404) [inline Music/music.md:28]: change sub-reddit to subreddit, change "requesting songs (most flacs)"  to "requesting lossless rips of albums"

File: Music/music.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ description: List of sources for getting all genres of Music.
...
+
+[**BlockTheSpot**](https://github.com/mrpond/BlockTheSpot) - Modifies Spotify to allow unlimited skips can remove the "Upgrade" button.
+
```
Comment (doggo404) [inline Music/music.md:31]: "Modified Spotify client to unlock unlock features without premium"

**Review by doggo404 — CHANGES_REQUESTED** [review]
god
help
us 
all

**Comment by Father-Dimi** [comment]
Alright, ill fix some stuff for it

**Comment by Father-Dimi** [comment]
also, do you guys think we should add crackedmultimc? and if we should put it in games or accounts?

**Comment by Father-Dimi** [comment]
updated music again due to rip request closing

**Comment by Helmasko** [comment]
All these have been implemented finally, if you have any other suggestions to add, make a new issue/pull request (and for gods sake dont make a whole new file to add 1 line)

**Pull request closed without merging**